CareKit/Connect/OCKConnectTableViewCell.m

```diff
@@ -101,12 +101,19 @@ - (void)updateView {
         _monogramLabel.text = nil;
     } else {
         _imageView.image = nil;
-        _monogramLabel.text = self.contact.monogram;
+        _imageView.backgroundColor = OCKSystemGrayColor();
+        
+        NSString *monogram = self.contact.monogram;
+        NSRange stringRange = {0, MIN([monogram length], 2)};
+        stringRange = [monogram rangeOfComposedCharacterSequencesForRange:stringRange];
+        NSString *clippedMonogram = [monogram substringWithRange:stringRange];
+        _monogramLabel.text = clippedMonogram;
```

Comment:
PR https://github.com/carekit-apple/CareKit/pull/10 takes care of the monogram clipping as part of `OCKContact`.